src/test/java/org/broadinstitute/hellbender/tools/sv/cluster/SVClusterEngineTest.java

```diff
@@ -400,15 +408,191 @@ public void testClusterTogetherCNVs() {
+                {"chr2", 1000, "chr2", 2000,
+                        GATKSVVCFConstants.ComplexVariantSubtype.delINV,
+                        Arrays.asList(
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.DEL, new SimpleInterval("chr1", 1100, 1500)),
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.INV, new SimpleInterval("chr1", 1600, 1900))
+                        ),
+                        false
+                },
+                // different coordinates
+                {"chr1", 1600, "chr1", 2400,
+                        GATKSVVCFConstants.ComplexVariantSubtype.delINV,
+                        Arrays.asList(
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.DEL, new SimpleInterval("chr1", 1100, 1500)),
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.INV, new SimpleInterval("chr1", 1600, 1900))
+                        ),
+                        false
+                },
+                // different subtypes
+                {"chr1", 1000, "chr1", 2000,
+                        GATKSVVCFConstants.ComplexVariantSubtype.delINVdel,
+                        Arrays.asList(
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.DEL, new SimpleInterval("chr1", 1100, 1500)),
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.INV, new SimpleInterval("chr1", 1600, 1900))
+                        ),
+                        false
+                },
+                // different number of intervals
+                {"chr1", 1000, "chr1", 2000,
+                        GATKSVVCFConstants.ComplexVariantSubtype.delINV,
+                        Arrays.asList(
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.DEL, new SimpleInterval("chr1", 1100, 1500))
+                        ),
+                        false
+                },
+                // different cpx intervals
+                {"chr1", 1000, "chr1", 2000,
+                        GATKSVVCFConstants.ComplexVariantSubtype.delINV,
+                        Arrays.asList(
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.DEL, new SimpleInterval("chr1", 800, 1100)),
+                                new SVCallRecord.ComplexEventInterval(GATKSVVCFConstants.StructuralVariantAnnotationType.INV, new SimpleInterval("chr1", 1600, 1900))
+                        ),
+                        false
+                },
+                // second cpx interval type is different
+                {"chr1", 1000, "chr1", 2000,
```

Comment:
I think you need to update the coordinates to match the intervals to make sure this is testing what you want it to, because the overall coordinates get checked before the CPX_INTERVALS

Reply:
Not sure I'm following - this record should match `cpx1` on all checks, including the POS/END overlap, except for the second CPX interval type. Or is there a problem that the POS/END don't match any CPX interval?

Reply:
I think I got turned around, this makes sense - you are forcing the coordinates to match cpx1 to make sure that the intervals get checked